src/app/server/src/server.ts

```diff
@@ -0,0 +1,48 @@
+import {Request, Response} from 'express';
+
+const express = require("express");
+const path = require("path");
+const fs = require("fs");
+const hbs = require("hbs");
+
+const app = express();
+const port = process.env.PORT || 3000;
+const rootDir = path.join(__dirname, "..");
```

Comment:
Is this always safe? Or better to have some sentinel root file to search for?

Reply:
Yeah, I think if this becomes an npm package as we discussed, the rootDir will be a parameter passed in to the entry point.